### ISSUE TITLE
Plans: Pro-rated credits notice should include a link to the public support page/worded differently. 

### ISSUE THREAD

**Issue by zdenys** [Automattic/wp-calypso#61293]
### What

The pro-rated credits notice should be improved and include a link to the public support page. This would be in line with this project: p58i-bTH-p2

### Why

The current message about pro-rated credits is not clear enough:

```
You have $X of pro-rated credits available from your current plan. Apply those credits towards an upgrade before they expire!
```

It's not clear which upgrade it refers to. We HEs know it's only applicable to plans. Some users are confused about this and reach out in chat and tickets to clear the confusion. And we can prevent that.

### How

We should at least guide users to read more about these credits at https://wordpress.com/en/support/manage-purchases/#pro-rated-credits 

![image](https://user-images.githubusercontent.com/39308239/154803504-ac7ebcc8-8b6a-44f5-bd56-6656baba6703.png)

That would reduce the number of interactions in chat and tickets about this subject.

**Comment by zdenys** [comment]
4793346-zd-woothemes

**Comment by Greatdane** [comment]
I agree, as there is always so much confusion with pro-rated credits!

**Comment by wiesenhauss** [comment]
I still see a lot of confusion regarding this. Some users will even ask to have those "credits" applied to other purchases (e.g., domains). At least two customers asked for a way to withdraw those to their bank accounts. I agree we have to include a link to a support page, but changing the words would be even better!

**Comment by zdenys** [comment]
35026147-hc

**Comment by masperber** [comment]
35483018-hc

**Comment by Greatdane** [comment]
Another confused user. I wish we could calculate how much time we have to spend weekly on answering questions from users 😢 
5226320-zd-woothemes

**Comment by jordesign** [comment]
@Automattic/editorial  is there a way we could word this more clearly to indicate that the upgrade credit is *just* for a higher plan? 

Maybe something like `You have $X of pro-rated credits available from your current plan. Apply those credits towards a higher plan before they expire!`

Would also appreciate your recommendation on how/where a link to the support document would be best located within the text (or after).

**Comment by SeywardDarby** [comment]
I would do this: "You have $X of pro-rated credits available to apply towards a higher plan. Apply them before they expire!"

**Comment by filipanoscampos** [comment]
Another confused user:
25123508-hc

**Comment by zdenys** [comment]
132350-hc

**Comment by zdenys** [comment]
Latest discussion here pdgrnI-24d-p2

**Comment by paxblueribbon** [comment]
Another confused and frustrated user
hc41670973
zd6047621

**Comment by jamiepalatnik** [comment]
Another report: 41365829-hc

**Comment by ddc22** [comment]
Should this be closed now :)
